.buildbot.sh

```diff
@@ -1,31 +1,48 @@
 #!/usr/bin/env bash
-# Build and extract bitcode from sqlite3 cross-compiling to CHERI
-# riscv64-purecap
+# Build and extract bitcode from lua cross-compiling to CHERI
+# riscv64-purecap and morello-purecap
 
 set -e
 
-if ! [ "$1" = "riscv64" ]; then
-	echo "Only riscv64 is supported."
+ARCH=
+CHERI=
+CONFIG_FLAGS=
+
+if [ "$1" = "morello-purecap" ]; then
+	ARCH="morello"
+	CHERI=$HOME/cheri/output/"$ARCH"-sdk
+	CONFIG_FLAGS="-target aarch64-unknown-freebsd13 \
+	--sysroot="$HOME"/cheri/output/rootfs-morello-purecap \
```

Comment:
Need to indent lines follow a `\`.

Reply:
Fixed (???) in [2314e76](https://github.com/capablevms/gllvm/pull/3/commits/2314e767e52ee235be3833c14ea1740427d62871). Let me know if I got this wrong, please.

Reply:
Too much (and too inconsistent) indent ;) Two spaces would fit in with the rest of the file.

Reply:
I am still not sure. In [9fccbc4](https://github.com/capablevms/gllvm/pull/3/commits/9fccbc4ef1bda10122c1b834d45ff721a45db617)?

Reply:
Fine!